lib/api_auth/base.rb

```diff
@@ -67,7 +69,7 @@ def generate_secret_key
 
     private
 
-    AUTH_HEADER_PATTERN = /APIAuth ([^:]+):(.+)$/
+    AUTH_HEADER_PATTERN = /APIAuth(-HMAC-(SHA[0-9]+))? ([^:]+):(.+)$/
```

Comment:
Also, should we enforce only matching on SHA digests? I'd be okay with that I think but others are supported by OpenSSL::Digest